### DISCUSSION TITLE
Kubernetes Support

### DISCUSSION THREAD

**Discussion by mehrabmp** [plausible/analytics#4348] [Self-Hosted Support]
Hi, Is it possible to add Kubernetes support to [Plausible Community Edition](https://plausible.io/blog/community-edition)
Thanks

**Comment by ruslandoga** [comment]
👋 @mehrabmp 

What do you mean by Kubernetes support?

**Reply by ruslandoga** [reply]
It's distributed as a container image so it can be used in Kubernetes already.

**Reply by mehrabmp** [reply]
I mean instructions or Helm charts for deploying to the k8s cluster

**Reply by mehrabmp** [reply]
I have seen the k8s deployment files in the project history that have been removed, but I don't know if they are still usable with the new Community version.
It would be great to have official support for k8s deployment.

**Reply by ruslandoga** [reply]
We used to have those but due to config rot and nobody to look after it (we don't use k8s internally) we had to remove it.

Relevant:
- https://github.com/plausible/community-edition/pull/71
- https://github.com/plausible/community-edition/pull/67#issuecomment-1514303962
- https://github.com/plausible/community-edition/pull/79
